### ISSUE TITLE
Check correctness for conv2d in NCHW format with preluActivationWeights

### ISSUE THREAD

**Issue by Linchenn** [tensorflow/tfjs#6354]
In WebGL's [Conv2d_impl.ts](https://github.com/tensorflow/tfjs/blob/fda7d7b1014ef89d61d6329e8048da0fec8ed03e/tfjs-backend-webgl/src/kernels/Conv2D_impl.ts), both [conv2dByMatMul](https://github.com/tensorflow/tfjs/blob/fda7d7b1014ef89d61d6329e8048da0fec8ed03e/tfjs-backend-webgl/src/kernels/Conv2D_impl.ts#L44) and [conv2dWithIm2Row](https://github.com/tensorflow/tfjs/blob/fda7d7b1014ef89d61d6329e8048da0fec8ed03e/tfjs-backend-webgl/src/kernels/Conv2D_impl.ts#L184) take `preluActivationWeights` as a parameter, while both functions do not consider the `NCHW format` for `preluActivationWeights`, which could be problematic for `NCHW format`.

`preluActivationWeights`, as defined in [fusedConv2d](https://github.com/tensorflow/tfjs/blob/fda7d7b1014ef89d61d6329e8048da0fec8ed03e/tfjs-core/src/ops/fused/conv2d.ts#L91), the shape could be similar with the input, which means it could have multiple dimensions and `dataFormat` may affect it:

>  @param preluActivationWeights Tensor of prelu weights to be applied as part of a `prelu` activation, typically the same shape as `x`.

In conclusion, we need to check:
1. The definition for the shape of `preluActivationWeights`. From my perspective, it should be compatible with the shape of the conv2d's output shape, instead of conv2d's  input shape, because the activation is applied after conv2d computation.
2. Should `dataFormat` describe the format of `preluActivationWeights`?

**Comment by Linchenn** [comment]
Currently, only CPU and WebGL supports fusedConv2d-NCHW. Fixed PReLU activation for them.
